content/hi/docs/home/supported-doc-versions.md

```diff
@@ -0,0 +1,11 @@
+---
+title: उपलब्ध प्रलेखन संस्करण
+content_type: custom
+layout: supported-versions
+card:
+  name: के बारे में
+  weight: 10
+  title: उपलब्ध प्रलेखन संस्करण
+---
+
+इस वेबसाइट में कुबेरनेट्स के वर्तमान और पिछले 4 संस्करणों के प्रलेखन हैं।
```

Comment:
The rest of this page seems to be missing. What happened here?

Reply:
Ah, no, that does look right (I forgot we'd automated it).

Reply:
Should I mark this conversation as "Resolved"?